core/include/detray/geometry/volume_connector.hpp

```diff
@@ -20,6 +20,7 @@ namespace detray {
 /// @param volume_grid [in] the indexed volume grid
 ///
 template <typename detector_t,
+          typename scalar_t /* detray::scalar */,
```

Comment:
It wasn't for documentation, my bad. I wanted to set it as the default

Reply:
From what I know this class is only used in `csv_io` so should we make the changes to this or not, provided this will not be used in the future. From here: https://github.com/acts-project/detray/issues/256#issuecomment-1125318656

Reply:
Let's fix this for now, so that we can decide later whether we can/need to salvage code from here for the geometry reading